docs/governance-strategy.md

```diff
@@ -0,0 +1,137 @@
+   - If all steering council members `Approve`, it may be merged any time.
+
+
+[^blocking]: When blocking any change or objecting to a proposal, provide a rationale for what must be changed and why you believe it is critically important. _Do not disapprove because of differences in opinion. Only disapprove if you have a major strategic concern_. See [Strategies for integrating objections](https://www.sociocracyforall.org/strategies-for-integrating-objections/) for what we are aiming for.
+
+
+### `executablebooks/myst-spec`: The MyST Specification
+
+`executablebooks/myst-spec` is the Source of Truth for the MyST Specification. It uses a combination of documentation, examples, schemas, tests, etc to define MyST markdown in a form that others could use to implement parsers and renderers. It is versioned and includes "releases" in order to make it easy for downstream implementors to track the changes they need to make.
+
+```{admonition} Implementation detail
+This repository will need to be modified as-needed to be the source of truth for the MyST Specification, and to have enough information to teach newcomers about its structure and function. One example for inspiration: [the Zarr specifications page](https://zarr.readthedocs.io/en/stable/spec.html).
+```
+
+The MyST Specification is changed through the MyST Enhancement Proposal process, described below.
+
+### `executablebooks/myst-eps`: MyST Enhancement proposals
+
+Because the MyST specification has many interior and exterior stakeholders, we use a more formal and structured process around changing the specification. This process is encoded in `executablebooks/myst-eps` and described briefly below
+
+1. **Open an issue in `myst-eps`** that describes the enhancement you'd like to make. The goal of the issue is to help others understand your idea and get informal alignment around it. _MEPs should ideally have at least two, and ideally 3-4 co-authors from different organizations._
+2. **Make a proposal via a PR to `myst-eps`**. Use a markdown template to structure your proposal. The `status` should initially be set to `Draft`. Here's a proposed template:
+   
+   ```
+   ---
+   mep:
+      id: <NNNN - increment last active MEP # by 1>
+      created: <yyyy-mm-dd - date MEP is active>
+      authors:
+        - <authors' real names, optional github handle>
+      status: <Draft | Active | Accepted | Not Accepted>
+      discussion: <URL of canonical location for discussion>
+   ---
+   # <title here>
+
+   ## Context
+   
+   <!-- provide context needed to understand this proposal. Describe the problem with MyST's current syntax or behavior. -->
+   
+   ## Proposal
+   
+   <!-- describe your proposed change to syntax in concrete terms. Include a layperson's description of this change if relevant. -->
+   
+   ## Examples
+   
+   <!-- provide examples of what this change would look like in the real world (e.g., raw MyST and rendered output). -->
+   
+   ## UX implications
```

Comment:
```suggestion
   ## UX Implications & Migration
```

I think calling out what migration is necessary should be in the header here, as it might be easy to overlook the description and also signals that we will support previous documents written in MyST over time.